packages/esm-patient-registration-app/src/config-schema.ts

```diff
@@ -179,6 +185,48 @@ export const esmPatientRegistrationSchema = {
         _description: 'The family/last name to record for unidentified patients.',
       },
     },
+    gender: {
+      _type: Type.Array,
+      _elements: {
+        value: {
+          _type: Type.String,
+          _description: 'The value for sex option',
```

Comment:
@makombe These are useless descriptions. Please provide better documentation in the future. I am fixing these.